### ISSUE TITLE
3.4.3 startup error

### ISSUE THREAD

**Issue by leasoncre** [Minecrell/ServerListPlus#84]
I get this everytime i boot up on 1.9 (didn't see a note on a specific version for spigot1.9
i tried deleteing the config.yml, no change

```
[22:30:24] [Server thread/INFO]: [ServerListPlus] Enabling ServerListPlus v3.4.3
[22:30:24] [Server thread/INFO]: [ServerListPlus] [Core] Reloading configuration...
[22:30:24] [Server thread/INFO]: [ServerListPlus] [Core] Saving configuration...
[22:30:24] [Server thread/WARN]: [ServerListPlus] [Core] Configuration is not enabled, nothing will be changed on the server! Please execute /slp enable to enable the configuration.
[22:30:24] [Server thread/ERROR]: [ServerListPlus] An internal error occurred while loading the core!
java.lang.NullPointerException
	at net.minecrell.serverlistplus.bukkit.BukkitPlugin.reloadCaches(BukkitPlugin.java:362) ~[ServerListPlus-3.4.3-Universal.jar:?]
	at net.minecrell.serverlistplus.core.ServerListPlusCore.reloadCaches(ServerListPlusCore.java:182) ~[ServerListPlus-3.4.3-Universal.jar:?]
	at net.minecrell.serverlistplus.core.ServerListPlusCore.reload(ServerListPlusCore.java:192) ~[ServerListPlus-3.4.3-Universal.jar:?]
	at net.minecrell.serverlistplus.core.ServerListPlusCore.<init>(ServerListPlusCore.java:135) ~[ServerListPlus-3.4.3-Universal.jar:?]
	at net.minecrell.serverlistplus.core.ServerListPlusCore.<init>(ServerListPlusCore.java:87) ~[ServerListPlus-3.4.3-Universal.jar:?]
	at net.minecrell.serverlistplus.bukkit.BukkitPlugin.onEnable(BukkitPlugin.java:133) [ServerListPlus-3.4.3-Universal.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:292) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:340) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at org.bukkit.craftbukkit.v1_9_R1.CraftServer.loadPlugin(CraftServer.java:361) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at org.bukkit.craftbukkit.v1_9_R1.CraftServer.enablePlugins(CraftServer.java:321) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at net.minecraft.server.v1_9_R1.MinecraftServer.t(MinecraftServer.java:411) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at net.minecraft.server.v1_9_R1.MinecraftServer.l(MinecraftServer.java:376) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at net.minecraft.server.v1_9_R1.MinecraftServer.a(MinecraftServer.java:331) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at net.minecraft.server.v1_9_R1.DedicatedServer.init(DedicatedServer.java:269) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at net.minecraft.server.v1_9_R1.MinecraftServer.run(MinecraftServer.java:527) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_74]
[22:30:24] [Server thread/INFO]: [ServerListPlus] Disabling ServerListPlus v3.4.3
[22:30:24] [Server thread/ERROR]: Error occurred while disabling ServerListPlus v3.4.3 (Is it up to date?)
java.lang.NullPointerException
	at net.minecrell.serverlistplus.bukkit.BukkitPlugin.onDisable(BukkitPlugin.java:151) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:294) ~[spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:364) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:424) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at net.minecrell.serverlistplus.bukkit.BukkitPluginBase.disablePlugin(BukkitPluginBase.java:46) [ServerListPlus-3.4.3-Universal.jar:?]
	at net.minecrell.serverlistplus.bukkit.BukkitPlugin.onEnable(BukkitPlugin.java:140) [ServerListPlus-3.4.3-Universal.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:292) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:340) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at org.bukkit.craftbukkit.v1_9_R1.CraftServer.loadPlugin(CraftServer.java:361) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at org.bukkit.craftbukkit.v1_9_R1.CraftServer.enablePlugins(CraftServer.java:321) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at net.minecraft.server.v1_9_R1.MinecraftServer.t(MinecraftServer.java:411) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at net.minecraft.server.v1_9_R1.MinecraftServer.l(MinecraftServer.java:376) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at net.minecraft.server.v1_9_R1.MinecraftServer.a(MinecraftServer.java:331) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at net.minecraft.server.v1_9_R1.DedicatedServer.init(DedicatedServer.java:269) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at net.minecraft.server.v1_9_R1.MinecraftServer.run(MinecraftServer.java:527) [spigot-1.9.jar:git-Spigot-f04e043-b3f6de0]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_74]
```

```
git-Spigot-f04e043-b3f6de0 (MC: 1.9) (Implementing API version 1.9-R0.1-SNAPSHOT)
ServerListPlus version 3.4.3

NOTE: 1.9 test server. i expect things to not work perfectly atm.
Plugins (50): ClearChat, MobEggSpawnerBlocker, SimpleTicketManager, HeadMap, VoxelSniper, TrophyHeads, Commandspy, Titlebar, AsyncWorldEditInjector, dynmap, VoteRoulette, OpenInv, Chairs, Votifier, ActionBarAPI, Multiverse-Core, ProtocolLib, LibsDisguises, AsyncWorldEdit, BlocksHub, GlobalMarket, PermissionsEx, mcMMO, ServerListPlus, Vault, HolographicDisplays, LWC, Multiverse-NetherPortals, WorldEdit, Essentials, Citizens, EssentialsChat, ColoredTablist, SuperVanish, EnjinMinecraftPlugin, Dynmap-Essentials, Sentry, WorldGuard, mcMMO-WorldGuard, BlocksHub-WorldGuard, MythicMobs, EssentialsProtect, ServerSigns, EssentialsSpawn, Prism, Dynmap-WorldGuard, Signs, BetonQuest, BlocksHub-Prism, AsyncWorldEdit-ActionBarAPI

**Comment by Minecrell** [comment]
Weird! Would you mind pasting the configuration it generates?

**Comment by leasoncre** [comment]
[ServerListPlus.txt](https://github.com/Minecrell/ServerListPlus/files/181216/ServerListPlus.txt)

**Comment by Minecrell** [comment]
That configuration says it's ServerListPlus v3.4.2 which makes everything even weirder. I'll test this later. Thanks for your report!

**Comment by leasoncre** [comment]
woops... wrong file (i run 2x 1.9 test servers, one is a live server copy, the other is a blank slate)
[ServerListPlus.txt](https://github.com/Minecrell/ServerListPlus/files/181217/ServerListPlus.txt)

**Comment by Minecrell** [comment]
Yeah, you're right thr config looks fine.

**Comment by Minecrell** [comment]
So is the server with 3.4.2 actually working, or same problem there?

**Comment by Minecrell** [comment]
From what I can tell, it only happens on the first start (when the configuration is generated). After that, the server is starting just fine for me.

**Comment by leasoncre** [comment]
no idea, never did anything with 3.4.2. [tested it, it loads fine, no configueing was done]

yeah, noticing that now too. only on first load

**Comment by Minecrell** [comment]
Alright, I fixed it. Will need to release a new version later today :|

**Comment by Minecrell** [comment]
I've released 3.4.4 which should fix that problem

**Comment by Carbonax** [comment]
I was having this issue on first start with 1.8.8 PaperSpigot too, thanks for fixing it
